ES/ESmy022.xml

```diff
@@ -358,7 +306,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
                                             <item xml:id="e3">
                                                <desc type="findingAid">Notes in the upper margin indicate
-                                                  indicate readings of <ref target="#p2_i1"/> assigned to the
+                                                  indicate readings of <ref target="ref=LIT5654MGessew"/> assigned to the
                                                   days of the week, the hours of the day and the feasts,
```

Comment:
clean `<ref type="work" target="LIT5654MGessew"/>` (you have a second ref inside of target element)

Reply:
Oh, sorry.